### ISSUE TITLE
Custom sender's email in email workflows

### ISSUE THREAD

**Issue by aasargsian** [calcom/cal.com#5794]
Is there a way to change the sender's email in email worflows? 

Using a familiar email address for email workflows is expected to substantially increase the open rate :)

We would like to use our email address as the sender's adress and a pre-set sender's name. Otherwise, our customers seem to be confused with Cal.Com in their inbox as they not used it and haven't subscribed to Cal.Com emails.

Tagging @alishaz-polymath since we discussed this on a call today :) I, unfortunately, can't add labels to the issue.

**Comment by CarinaWolli** [comment]
Hi @aasargsian, we could make the name of the sender custom. So it would be the same we have it for SMS actions where you can define your custom sender ID. It still will be sent from hello@cal.com but your custom name will be shown and the reply-to email will also be your email from your Cal account. Would that solve your issue?

**Comment by aasargsian** [comment]
@CarinaWolli yes, that would solve the issue that we are facing. Is this possible in the current version or it needs development?

**Comment by CarinaWolli** [comment]
@aasargsian Great! It's not yet possible in the current version, but I will work on it
